migrations/sqls/20190324132907-dbdev-up.sql

```diff
@@ -0,0 +1,185 @@
+/* Replace with your SQL commands */
+/*
+Create users tables: I figured that later on, this application can be scaled to mainain, volunteers, board members, and maybe even donors and staff (HR management) 
+*/
+CREATE TABLE IF NOT EXISTS users (
+  userId int AUTO_INCREMENT PRIMARY KEY,
+  firstName varchar(64) NOT NULL,
+  lastName varchar(64) NOT NULL,
+  accessLevel ENUM('Administrator','Manager','Staff'), 
+  userName nvarchar(64) NOT NULL,
+  password nvarchar(64) NOT NULL
+);
+INSERT INTO users(firstName,lastName,accessLevel,userName,password)
+VALUES('Leonel','Nieto','Administrator','leonelnieto','password');
+
+/*Create person.person table */
+CREATE TABLE IF NOT EXISTS person (
+  personId int AUTO_INCREMENT PRIMARY KEY,
+  firstName nvarchar(255) NOT NULL,
+  lastName nvarchar(255) NOT NULL,
+  dob date,
+  gender ENUM('Male','Female','Non-Binary','Other'),  /*Gender constraint at front end */
+  dateAdded DATETIME DEFAULT CURRENT_TIMESTAMP,
+  dateModified DATETIME DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  addedBy int NOT NULL /*Self Reference to Person ID*/,
+  modifiedBy int NOT NULL /*Self Reference to Person ID*/,
+  FOREIGN KEY (addedBy) REFERENCES person(personId),
+  FOREIGN KEY (modifiedBy) REFERENCES users(userId)
+);
+/*Insert test data into person table*/
+INSERT INTO person(firstname,lastname,dob,gender,addedby,modifiedby) VALUES (
```

Comment:
Same here -- this shouldn't be in the migration. We should insert this into the DB either manually or via an API request done in the backend server.